statsmodels/tsa/stattools.py

```diff
@@ -1055,6 +1055,115 @@ def arma_order_select_ic(y, max_ar=4, max_ma=2, ic='bic', trend='c',
     return Bunch(**res)
 
 
+def kpss(x, null_hypo="level", lags=None):
+    """
+    Kwiatkowski-Phillips-Schmidt-Shin test for stationarity.
+
+    Computes the Kwiatkowski-Phillips-Schmidt-Shin (KPSS) test for the null
+    hypothesis that x is level or trend stationary.
+
+    Parameters
+    ----------
+    x : array_like, 1d
+        Data series
+    null_hypo : str{"level", "trend"}
+        Indicates the null hypothesis for the KPSS test
+        * "level" : The data is level stationary (default)
+        * "trend" : The data is trend stationary
+    lags : int
+        Indicates the number of lags to be used. If None (default),
+        lags is set to int(12 * (n / 100) ** (1 / 4)), as outlined in
+        Schwert (1989).
+
+    Returns
+    -------
+    kpss_stat : float
+        The KPSS test statistic
+    p_value : float
+        The p-value of the test
```

Comment:
This needs a note in the current version that pvalues are truncated, i.e. report boundary points if pvalue < 0.01 or > 0.1, and interpolates.